File: Meta 2.2/src/main/java/Application.java
import javax.swing.*;
import java.awt.*;

/**
 * Contiene con JPanel y las propiedades de la ventana a crear
 * @author <a href="https://github.com/medina1402" target="_blank">Abraham Medina Carrillo</a>
 */

public class Application extends JFrame {
    public Home home;

    public Application() {
        setTitle("MiniGame");
        Dimension sizeScreen = Toolkit.getDefaultToolkit().getScreenSize();
        setSize((int) (sizeScreen.height * 0.75), (int) (sizeScreen.height * 0.95));

        home = new Home(this);
        setContentPane(home);

        setLocationRelativeTo(null);
        setVisible(true);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
    }

    public static void main(String[] args) {
        new Application();
    }
}

File: Meta 4.2/src/main/java/Application.kt
import utils.Vector
import gui.Container
import javax.swing.JFrame

/**
 * Clase principal, tiene las propiedades del JFrame (hereda de el)
 * @author <a href="https://github.com/medina1402" target="_blank">Abraham Medina Carrillo</a>
 */
class Application: JFrame() {
    private var container: Container

    init {
        layout = null
        setSize(800, 500)
        isResizable = false
        setLocationRelativeTo(null)
        title = "Uso de concurrencia"
        defaultCloseOperation = EXIT_ON_CLOSE

        container = Container(Vector(0, 0), Vector(width, height))
        add(container)
        getRootPane().contentPane = container
    }
}

/**
 * Funcion principal de ejecucion de la Aplicacion
 */
fun main() {
    Application().isVisible = true
}